src/cli.ts

```diff
@@ -40,5 +58,5 @@ Publish package versions from one registry to another.`)
   .argv
 
 // Publish all versions of the specified package
-require('./index').sync(name, from, to, dryRun)
+require('./index').sync(name, from, to, dryRun, latestOnly, latestMajors, repository)
```

Comment:
Codacy has a fix for the issue: [Replace `'./index')` with `"./index")⏎··`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263163)
```suggestion
require("./index")
```